tool/tsh/common/tsh.go

```diff
@@ -3232,23 +3233,39 @@ func serializeSessions(sessions []types.SessionTracker, format string, w io.Writ
 }
 
 func printSessions(output io.Writer, sessions []types.SessionTracker) {
-	table := asciitable.MakeTable([]string{"ID", "Kind", "Created", "Hostname", "Address", "Login", "Command"})
+	table := asciitable.MakeTable([]string{"ID", "Kind", "Created", "User", "Target", "Address"})
```

Comment:
Fair point, but I really don't want to add another column. The usability of these tables quickly approaches 0 as they get too wide.

I wonder if we can stuff all this information in the Target and Address columns.

Maybe "Target" is the exec command and address is namespace/pod@cluster?

Reply:
We had several clients who complained about the lack of visibility in our moderated sessions because they could only understand what command to execute after the moderator had entered the session and at that point, it had already started. The fact that we omit or are not clear about the commands causes insecurity for those moderating the session since they have no idea what will actually be executed.

If we can clarify what 'target' means for each session type, I don't think it will contradict customer expectations, such as:
- SSH: Shell for tiago@zirkon
- Kube: Exec 'rm -rf /'
- Windows: Session for tiago@windows